# Structure for packets sent from the Program Node to the Motion Node

float32 x       # x coordinate in mm
float32 y       # y coordinate in mm

---

string status # location status, should include or change this to an int?